test/test_integration.py

```diff
@@ -96,7 +143,7 @@
 csv_latin1_encoding = {
     'name': 'csv_latin1_encoding',
     'raw_data': ['a,b,c',
-                 u'1,2,4L�ve',
+                 u'1,2,4Löve',
```

Comment:
We should not be able to see any changes in this line.  Your text editor/IDE is encoding the values and it changes them. I recommend using PyCharm, it could show you the changed history if you select a file and right click, the options are `Local History` - > `Show History.` You can also have the history using Git just under Local History.

The values look similar but they are different in terms of bit representation